[2024-09-19T10:45:39.894+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-19T10:45:40.003+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.second_task scheduled__2024-09-14T00:00:00+00:00 [queued]>
[2024-09-19T10:45:40.014+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.second_task scheduled__2024-09-14T00:00:00+00:00 [queued]>
[2024-09-19T10:45:40.015+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-19T10:45:40.047+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2024-09-14 00:00:00+00:00
[2024-09-19T10:45:40.072+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=556) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-19T10:45:40.102+0000] {standard_task_runner.py:72} INFO - Started process 574 to run task
[2024-09-19T10:45:40.092+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'second_task', 'scheduled__2024-09-14T00:00:00+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmplsmh8vdo']
[2024-09-19T10:45:40.120+0000] {standard_task_runner.py:105} INFO - Job 169: Subtask second_task
[2024-09-19T10:45:40.403+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.second_task scheduled__2024-09-14T00:00:00+00:00 [running]> on host 77aa4aabff6e
[2024-09-19T10:45:41.068+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-14T00:00:00+00:00'
[2024-09-19T10:45:41.131+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-19T10:45:41.172+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-19T10:45:41.191+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey am task 2 and will be running after task 1']
[2024-09-19T10:45:41.275+0000] {subprocess.py:86} INFO - Output:
[2024-09-19T10:45:41.305+0000] {subprocess.py:93} INFO - hey am task 2 and will be running after task 1
[2024-09-19T10:45:41.307+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-19T10:45:41.593+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-19T10:45:41.603+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=second_task, run_id=scheduled__2024-09-14T00:00:00+00:00, execution_date=20240914T000000, start_date=20240919T104540, end_date=20240919T104541
[2024-09-19T10:45:41.681+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-19T10:45:41.897+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-19T10:45:41.907+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
